services/invitations/src/simcore_service_invitations/services/invitations.py

```diff
@@ -13,35 +12,30 @@
 
 _logger = logging.getLogger(__name__)
 
-#
-# Errors
-#
-
 
 class InvalidInvitationCodeError(Exception):
     ...
 
 
-#
-# Models
-#
-
-
 class _ContentWithShortNames(InvitationContent):
     """Helper model to serialize/deserialize to json using shorter field names"""
 
     @classmethod
-    def serialize(cls, model_data: InvitationContent) -> str:
+    def serialize(cls, model_obj: InvitationContent) -> str:
         """Exports to json using *short* aliases and values in order to produce shorter codes"""
         model_w_short_aliases_json: str = cls.construct(
-            **model_data.dict(exclude_unset=True)
+            **model_obj.dict(exclude_unset=True)
         ).json(exclude_unset=True, by_alias=True)
+        # NOTE: json arguments try to minimize the amount of data
+        # serialized. The CONS is that it relies on models in the code
+        # that might change over time. This might lead to some datasets in codes
+        # that fail in deserialization
```

Comment:
I think the code is clear at what it does. The keys in the json are shortened.
I understand form you comment that these keys can change over time and when the message is deserialised back it can fail.

How does my below summary sound?
```suggestion
        # NOTE: if the short-version keys in the output json change over time
        # there can be a failure during deserialisation.
```